instrumentation/servlet/servlet-3.1/src/test/java/io/opentelemetry/instrumentation/hypertrace/servlet/v3_1/Servlet31InstrumentationTest.java

```diff
@@ -79,6 +91,58 @@ public void postJson() throws Exception {
         spanData
             .getAttributes()
             .get(HypertraceSemanticAttributes.httpResponseHeader(TestServlet.RESPONSE_HEADER)));
-    server.stop();
+  }
+
+  @Test
+  public void requestCast() throws Exception {
```

Comment:
@davexroth @surajpuvvada here is an example of the cast exception. The first request fails, however the second passes but the attributes that are captured via wrappers are omitted. This is the best what we can do.